src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -186,5 +188,38 @@ public static void verifyColumnSort(String columnName, String tableName, String
             assertTrue(expectedResult, getElementAsTable(tableName).verifyColumnCellsAreSortedDescending(columnName));
         }
     }
+    
+    /**
+     * Verifies that a specific cell, given by the row and column, in the table contains the given text.
+     * <p>
+     * <b>Gherkin Examples:</b>
+     * <ul>
+     * <li>I verify the cell in the last row and the Employee First Name column of the Employee table does not contain the text Alice</li>
+     * <li>I verify the cell in the 2nd row and the Employee First Name column of the Employee table contains the text Bob</li>
+     * </ul>
+     * @param rowNum String the row number. Can be "la" to specify the last row, or an integer.
+     * @param columnName String the name of the column to verify
+     * @param tableName String the name of the table containing the column
+     * @param assertion String if null is passed, looks for match(es), if any strong value is passed, looks for the value to not exist.
+     * @param textToMatch String the text to look for in the column
+     */
+    @Then("^I verify the cell in the (la|\\d+)(?:st|nd|rd|th) row and the (.*) column of the (.*?)( do(?:es)? not)? contains? the text (.*?)$")
+    public static void verifyCellInSpecifiedRow(String rowNum, String columnName, String tableName, String assertion, String textToMatch) {
+    	boolean negate = !StringUtils.isEmpty(assertion);
+    	
+    	var table = (Table)getElement(tableName);
```

Comment:
The method for returning this wraps the error if it can't be cast, but I'm starting to wonder if we should instead get rid of that code and use the way you've done it here. If you feel strongly about the way you have this here we can discuss it.
```suggestion
    	var table = getElementAsTable(tableName);
```